tests/rptest/tests/redpanda_test.py

```diff
@@ -49,3 +48,8 @@ def topic(self):
 
     def setUp(self):
         self.redpanda.start()
+
+    def resource_dir(self):
+        dir_path = os.path.dirname(os.path.realpath(__file__))
+        resource_dir_path = os.path.join(dir_path, "..", "resources")
+        return os.path.abspath(resource_dir_path)
```

Comment:
@ivotron please review

Reply:
lgtm. An alternative would be to use the built-in templating rendering from ducktape, like the one used [here](https://github.com/vectorizedio/redpanda/blob/dev/tests/rptest/services/redpanda.py#L156). Although the JS script is not a proper template, so maybe it should be left in resources...

the other alternative would be to use the artifact packaging feature of pip (see [here](https://kiwidamien.github.io/making-a-python-package-vi-including-data-files.html)), and then use the python stdlib `pkg_resources` module.

Reply:
Oh nice, I didn't know about the templating stuff. What I plan on doing was to do some templating stuff to create scripts on the fly, to be able to dynamically create a script with custom input/output topics, etc. When I move to that, I can remove this resource_dir.